WalletWasabi.Fluent/ViewModels/Wallets/Home/History/Details/CoinJoinsDetailsViewModel.cs

```diff
@@ -63,6 +58,8 @@ private void Update()
 			TransactionId = transaction.Id;
 			TransactionIds = new ObservableCollection<uint256>(transaction.Children.Select(x => x.Id));
 			TxCount = TransactionIds.Count;
+			ConfirmationTime = _wallet.Transactions.TryEstimateConfirmationTime(transaction.Id);
```

Comment:
This is wrong. You need the confirmation time of the child that will confirm the last.

Reply:
This is the same as it's on `master`. It only got moved down to the `Update` function. If it's wrong here, then it's also wrong in `master`.

Reply:
It turned out it was mistakenly added with [this commit](https://github.com/zkSNACKs/WalletWasabi/commit/c87e5fbb94e64481d6e8458f805668617ce50ccc).
You can remove it, and every other property too that was needed for it.